NEWS.md

```diff
@@ -39,6 +56,11 @@ changes:
   * Added support for FIPS 186-5 deterministic ECDSA signature
     generation to the FIPS provider.
 
+  * Deprecated `EVP_PKEY_ASN1_METHOD` related functions.
+
+OpenSSL 3.5
+-----------
+
 ### Major changes between OpenSSL 3.5.0 and OpenSSL 3.5.1 [under development]
```

Comment:
This date DOES need updating. The release scripts only update the date at the top of the file. This file should essentially be the same as the 3.5 CHANGES.md up until the most recent 3.5 release below here. Once 3.6 final is out we no longer update this section, but until then we should do

Reply:
Yes, more or less. I would probably only include the changes up until 3.5.2 since 3.5.3 has not actually been released yet. We will evetually need to update this with any additional 3.5.x releases that are made before the final 3.6 release is made - but not for the 3.6 alpha.

We should do similar updates in the CHANGES.md file.